app/actions/views/channel.js

```diff
@@ -401,14 +401,19 @@ export function handleSelectChannel(channelId, fromPushNotification = false) {
 }
 
 export function handleSelectChannelByName(channelName, teamName) {
-    return async (dispatch, getState) => {
+    return async (dispatch, getState) => { // eslint-disable-line consistent-return
```

Comment:
for actions  we should be returning `{data: ...}` or `{error:...}` and the return should be consistent.

Reply:
Done.